### PR TITLE
Template resolver helpers

### PR DESCRIPTION
**Pull request by MartinMystikJonas**

### CODE REVIEW

File: src/LatteContext/LatteContext.php [hunk 1]
```diff
@@ -115,6 +115,16 @@ public function templateRenderFinder(): TemplateRenderFinder
         return $this->templateRenderFinder;
     }
 
+    public function getClassTemplateContext(string $className): TemplateContext
+    {
+        return new TemplateContext(
+            $this->variableFinder()->find($className),
```
Comment (lulco) [inline src/LatteContext/LatteContext.php:121]: this returns only "global" class variables, components, forms and filters right?

Comment (lulco) [reply src/LatteContext/LatteContext.php:121]: can ClassTemplate have form, component, filter? I don't think so

Comment (MartinMystikJonas) [reply src/LatteContext/LatteContext.php:121]: Components/forms from createComponent are registered as global class.

Everything can be global class from annotation

Comment (lulco) [reply src/LatteContext/LatteContext.php:121]: I ment something else. If mabar has TemplateClasses and pass them to this method he will not get forms, components and filters, only variables.

**Review by lulco — APPROVED** [review]

File: src/LatteContext/LatteContext.php (hunk 1 above)
Comment (MartinMystikJonas) [reply src/LatteContext/LatteContext.php:121]: He going to extract control class from template class and then he will pass control class here. To extract variables from template class LatzeContextHelper is used.

**Review by lulco — APPROVED** [review]

**Pull request merged**